airbyte-integrations/connectors/destination-pinecone/unit_tests/pinecone_indexer_test.py

```diff
@@ -175,7 +184,7 @@ def generate_catalog():
 def test_pinecone_pre_sync(mock_describe_index):
     indexer = create_pinecone_indexer()
     indexer.pre_sync(generate_catalog())
-    indexer.pinecone_index.delete.assert_called_with(filter={"_ab_stream": "example_stream2"})
+    indexer.pinecone_index.delete.assert_called_with(filter={"_ab_stream": "ns2_example_stream2"}, namespace="ns2")
```

Comment:
Just to double check - do you think think w should this as a breaking change? 

Namely, does the namespace come through from the source by default, and would this change the destination index name? Or is this purely opt-in, so we can assume that wherever it was not handled previously, this is 'fixing' the prior omitted prefix?

Reply:
It's not a breaking change because the namespace is opt-in (can be configured on the connection level) - it changes here because I added a namespace to the test.